### ISSUE TITLE
Sentinel Docs > Replica selection and priority: Wrong mention of run ID instead of replica-priority

### ISSUE THREAD

**Issue by karuppiah7890** [redis/redis-doc#1627]
In `Sentinel` Docs under `Replica selection and priority` section - https://redis.io/topics/sentinel#replica-selection-and-priority , it says -

```
Redis masters (that may be turned into replicas after a failover), and replicas, all must be configured with a replica-priority if there are machines to be strongly preferred. Otherwise all the instances can run with the default run ID (which is the suggested setup, since it is far more interesting to select the replica by replication offset).
```

Note the `Otherwise all the instances can run with the default run ID`. I think it shouldn't be `default run ID`. I think it should be `default replica-priority`

I can raise  a PR for fixing this

**Comment by yoav-steinberg** [comment]
@yossigo, @hwware can you take a look ^.

**Comment by hwware** [comment]
@karuppiah7890  I think your understanding is: If There is no replica-priority config in the sentinel.conf for redis master and replicas,  then all the instances can run with the default replica-priority (which is 100), right?   
@yossigo  Do you  agree with my thought?

**Comment by karuppiah7890** [comment]
I don't know the value of the default replica priority 😅 but the doc section is trying to talk about that I believe, given offset is the next and important criteria after replica priority as part of the docs

**Comment by yossigo** [comment]
@karuppiah7890 @hwware I think you're both right.
I'm not sure if I'm getting the original intention of this paragraph, but assuming I do I'd rephrase it as follows:

> In most cases, `replica-priority` does not need to be set explicitly so all instances will use the same default value. If there is a particular fail-over preference, `replica-priority` must be set on *all instances, including masters*, as a master may become a replica at some future point in time - and it will then need the proper `replica-priority` settings.

Does that make sense and address the issue?

**Comment by karuppiah7890** [comment]
@yossigo I think that makes sense! 👍 Do we also add that when replica priority is the default, then offset is considered next for replica selection? Or it's just redundant I guess? As the previous parts talk about how the selection happens

**Comment by hwware** [comment]
> @yossigo I think that makes sense! 👍 Do we also add that when replica priority is the default, then offset is considered next for replica selection? Or it's just redundant I guess? As the previous parts talk about how the selection happens

I think you do not need to add this, because the document already write:

The replicas are sorted by replica-priority as configured in the redis.conf file of the Redis instance. A lower priority will be preferred.
If the priority is the same, the replication offset processed by the replica is checked, and the replica that received more data from the master is selected.

If you check the sentinel.conf, you could find the default replica-priority is 100
